xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
+     </para>
     </step>
     <step>
-     <para>On &node1;, proceed as follows:</para>
+     <para>On <systemitem class="domainname"
+      >&node4;</systemitem>, proceed as follows:</para>
      <substeps>
-      <!--
       <step>
        <para>In the firewall settings, open the TCP port 3121 for
         &pmremote;.</para>
       </step>
-      -->
+      <step>
+       <para>Install the <package>pacemaker-remote</package> and
+         <package>crmsh</package> packages: </para>
+       <screen>&prompt.root;<command>zypper</command> in pacemaker-remote crmsh</screen>
+      </step>
+      <step>
+       <para>Enable and start the &pmremote; service on <systemitem
+         class="domainname">&node3;</systemitem>:</para>
+       <screen>&prompt.root;<command>systemctl</command> enable pacemaker_remote
+&prompt.root;<command>systemctl</command> start pacemaker_remote</screen>
+      </step>
+      <step>
+       <para>Create the <filename>/etc/pacemaker</filename> directory and
+        change its group:</para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0750 /etc/pacemaker
+&prompt.root;<command>chgrp</command> haclient /etc/pacemaker</screen>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>On <systemitem class="domainname">&node3;</systemitem>,
+      run <command>csync2</command> to copy the authentication key from
+      your physical host to your guest node:</para>
+     <screen>&prompt.root;<command>csync</command> -xv</screen>
+    </step>
+    <step>
+     <para>
+      To verify the host connection to the guest, run <command>ssh</command>
+      on one of the cluster nodes. This connection will fail, but
+      <emphasis>how</emphasis> it fails gives you hints about its
+      connectivity:
+     </para>
+     <screen>&prompt.root;<command>ssh</command> -p 3121 &node4;</screen>
+     <para>From the output you can tell if your connection works:</para>
+     <variablelist>
+      <varlistentry>
+       <term>Working connection</term>
+       <listitem>
+        <screen>ssh_exhange_identification: read: Connection reset by peer.</screen>
+       </listitem>
+      </varlistentry>
+      <varlistentry>
+       <term>Broken connection</term>
+       <listitem>
+        <screen>ssh: connect to host &node4; port 3121: No route to host
+ssh: connect to host &node4; port 3121: Connection refused</screen>
+        <para>If you see either of the previous messages, the connection does
+        not work. Use the <option>-v</option> option for <command>ssh</command>
+         and execute the command again to see debugging messages. This could
+         be helpful to find connection, authentication, or configuration
+         problems. Multiple <option>-v</option> options increase the verbosity.
+        </para>
+       </listitem>
+      </varlistentry>
+     </variablelist>
+     <para>If needed, add more guest nodes and configure them as described above.</para>
```

Comment:
If needed, add more physical remote nodes and configure them as described above.